doc/api/assert.md

```diff
@@ -2448,7 +2445,7 @@ assert.throws(throwingFirst, /Second$/);
 Due to the confusing error-prone notation, avoid a string as the second
 argument.
 
-[Abstract Equality Comparison]: https://tc39.github.io/ecma262/#sec-abstract-equality-comparison
+[Abstract Equality Comparison]: https://262.ecma-international.org/#sec-abstract-equality-comparison
```

Comment:
> The document at https://tc39.es/ecma262/ is the most accurate and up-to-date ECMAScript specification.

Source: https://262.ecma-international.org/#metadata-block

```suggestion
[Abstract Equality Comparison]: https://tc39.es/ecma262/#sec-abstract-equality-comparison
```

Reply:
> > The document at https://tc39.es/ecma262/ is the most accurate and up-to-date ECMAScript specification.
> 
> Source: https://262.ecma-international.org/#metadata-block

I changed it to the other document because the one that was linked originally (which is the one you are suggesting here) no longer uses the term _Abstract Equality Comparison_ as far as I can tell. (It uses _IsLooselyEqual_.)

Linking to the spec and using somewhat jargon-y terms like _Abstract Equality Comparison_ is unlikely to be useful compared to something straightforward like "uses the `==` operator". What do you think of that change instead throughout the doc?

Reply:
Changed happen in https://github.com/tc39/ecma262/pull/2378/files. If the spec doesn't use this anymore, +1 from me to drop it in our docs as well. "uses the `==` operator" SGTM.

Reply:
> If the spec doesn't use this anymore, +1 from me to drop it in our docs as well. "uses the `==` operator" SGTM.

Done!